storage/storagedriver/zfs-driver.md

```diff
@@ -263,7 +263,7 @@ There are several factors that influence the performance of Docker using the
   filesystems like ZFS. ZFS mitigates this by using a small block size of 128k.
   The ZFS intent log (ZIL) and the coalescing of writes (delayed writes) also
   help to reduce fragmentation. You can monitor fragmentation using
-  `zfs status`. However, there is no way to defragment ZFS without reformatting
+  `zpool list`. However, there is no way to defragment ZFS without reformatting
```

Comment:
Wondering if the intent was to use `zpool status`?

Reply:
Yeah, unfortunately, I mistyped. Shame on me. :)

Reply:
I changed it @thaJeztah . Is this good to go?

Reply:
Thank you, @TheHades and @thaJeztah